test/NuGet.Clients.Tests/NuGet.VisualStudio.Internal.Contracts.Test/Formatters/VersionInfoContextInfoFormatterTests.cs

```diff
@@ -30,6 +30,7 @@ public void SerializeThenDeserialize_WithValidArguments_RoundTrips(VersionInfoCo
 
             Assert.Equal(expectedResult.DownloadCount, actualResult.DownloadCount);
             Assert.Equal(expectedResult.Version, actualResult.Version);
+            // TODO: FAILING TEST: actualResult.PackageDeprecationMetadata is null, so we nullref here. 4 props appear to be written, but when reading, only 3 props are read.
```

Comment:
@dtivel  - this is that failing test. would love a tip on how to attack.

Reply:
@rrelyea, the bug is that you added 2 new properties to the PackageSearchMetadataContextInfoFormatter but you forgot to increment the total property count by 2 [here](https://github.com/NuGet/NuGet.Client/blob/1c52baffa1b24223b75149a00bbc7229db1dd6ef/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Formatters/PackageSearchMetadataContextInfoFormatter.cs#L217).

Reply:
fixed in latest iteration.;